### ISSUE TITLE
Deprecation warning to fix

### ISSUE THREAD

**Issue by vallsv** [silx-kit/pyFAI#1126]
It would be good to fix it for the workshop.

```
WARNING:pyFAI.DEPRECATION:Function _integrate2d_legacy is deprecated since pyFAI version 0.18.
  File "/home/valls/.local/lib/python3.5/site-packages/pyFAI/worker.py", line 382, in process
    integrated_result = self.ai.integrate2d(**kwarg)
```

**Comment by vallsv** [comment]
Have to ask @kif what to do

**Comment by vallsv** [comment]
I think it was fixed by Jerome with 5fb349867c8098f72146ccd494ab326764442a82
